File: codewars/7kyu/two-oldest-ages/dduck-two-oldest-ages.js
// https://www.codewars.com/kata/511f11d355fe575d2c000001/train/javascript
const twoOldestAges = (ages) => ages.sort((a, b) => a - b).slice(-2);

twoOldestAges([1, 2, 10, 8]);
//?

File: codewars/7kyu/two-oldest-ages/dduck-two-oldest-ages.py
# https://www.codewars.com/kata/511f11d355fe575d2c000001/

from unittest import TestCase


two_oldest_ages = lambda ages: sorted(ages)[-2::]


TestCase().assertEqual(two_oldest_ages([1, 5, 87, 45, 8, 8]), [45, 87])
TestCase().assertEqual(two_oldest_ages([6, 5, 83, 5, 3, 18]), [18, 83])
TestCase().assertEqual(two_oldest_ages([10, 1]), [1, 10])
